### PR TITLE
Feature/future term teaching card

### PR DESCRIPTION
**Pull request by fanglinfang**
https://trello.com/c/ugnnc5W9

Test cases are on the bottom of the Spec: https://docs.google.com/document/d/100rh6F2H1Xg4FmyLGG6aNjqWbY-z_QbpAm-qAgNLrmY/edit#

Summary of changes specific to this PR:

New Future quarter cards appears to instructors who are teaching courses in upcoming future quarter
Future Quarter card uses same template as summary schedule card, with logic displaying content specific to future quarter
User can navigate to future quarter course sections in teaching page
User can navigate directly to specific future course from future quarter card on homepage.
Adds a link to view the future quarter's important dates and deadlines in academic calendar
Adds enrollment count to future quarter card. If students enrolled, link to view class list. 
User sees date when future quarter starts and how many courses they are teaching

### CODE REVIEW

**Comment by klpittman** [comment]
@bexar Functionality looks good. What are your thoughts on the order of the cards? Currently on my build it goes:

1. Spring 2013 Teaching Schedule
1. Summer 2013 Teaching Schedule
1. Spring 2013 Schedule (visual schedule)

I guess I'm wondering if quarters should stay together and push the Summer to the bottom? How does student side handle it?

**Comment by klpittman** [comment]
myuw__home

@bexar Also begs the question of whether the Important Dates & Deadlines link should be in the current quarter schedule card?

**Comment by fanglinfang** [comment]
i'd want to see the cards of the same quarter stay together

**Comment by bexar** [comment]
Yes, cards of the same quarter should stay together. The future quarter card can be pushed to the bottom (this is how it's documented on the spec). I believe student side handles it the same way.

**Comment by bexar** [comment]
@klpittman I suppose I can't think of a good reason why dates and deadlines link should not also appear on current quarter schedule card.

File: myuw/templates/handlebars/card/summary/schedule.html [hunk 1]
```diff
@@ -59,7 +59,7 @@ <h4 class="myuw-card-row-heading" style="margin-bottom:0px;"><div class="c{{colo
                   </div>
               </div>
             {{/each}}
-            {{#if future_term}}<div class="future-quarter-links" style="margin-top:16px;"><a href="/academic_calendar/#{{term.year}},{{term.quarter}}">View {{capitalizeString quarter}} {{year}} important dates and deadlines.</a></div>{{/if}}
+            <div class="future-quarter-links" style="margin-top:16px;"><a href="/academic_calendar/#{{term.year}},{{term.quarter}}">View {{capitalizeString quarter}} {{year}} important dates and deadlines.</a></div>
```
Comment (fanglinfang) [inline myuw/templates/handlebars/card/summary/schedule.html]: {{term.year}},{{term.quarter}} not work for current quarter

**Pull request merged**